index.js

```diff
@@ -63,7 +79,7 @@ function loadFragment(event) {
   if (!src) return
 
   const loader = menu.querySelector('include-fragment')
-  if (loader) {
+  if (loader && !loader.hasAttribute('src')) {
```

Comment:
Just curious, what made you add this condition?

Reply:
If the fetch triggered by `mouseover` is still running when `<details>` is clicked, this prevents the `toggle` event from requesting a second fetch. The `<include-fragment>` element should already ignore it, because the `src` value is the same, so this is just an extra precaution.